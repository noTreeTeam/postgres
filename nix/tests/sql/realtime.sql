-- only a publication from supabase realtime is expected
SELECT
    pubname AS publication_name,
    pubowner::regrole AS owner,
    puballtables,
    pubinsert,
    pubupdate,
    pubdelete,
    pubtruncate
FROM
    pg_publication;
