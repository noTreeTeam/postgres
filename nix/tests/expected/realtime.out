-- only a publication from supabase realtime is expected
SELECT
    pubname AS publication_name,
    pubowner::regrole AS owner,
    puballtables,
    pubinsert,
    pubupdate,
    pubdelete,
    pubtruncate
FROM
    pg_publication;
 publication_name  |  owner   | puballtables | pubinsert | pubupdate | pubdelete | pubtruncate 
-------------------+----------+--------------+-----------+-----------+-----------+-------------
 supabase_realtime | postgres | f            | t         | t         | t         | t
(1 row)

